sql_metadb/derived_tables/agreements_subscription_agreement_org_ext.sql

```diff
@@ -8,14 +8,15 @@ SELECT
     sao.sao_owner_fk AS subscription_agreement_id,
     sao.sao_org_fk AS sao_org_id,
     org.org_name AS sao_org_name,
-    sao.sao_role AS sao_role_id,
-    saor.rdv_value AS sao_role_value,
-    saor.rdv_label AS sao_role_label,
+	saor.saor_role_fk AS sao_role_id,
```

Comment:
Change data type to uuid? @nassibnassar would have to answer that. In general, there shouldn't be a problem within agreements when you connect data. I'm just not sure how it's supposed to be in the metadb with the IDs.